src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -121,14 +116,22 @@ ApplicationManagerImpl::ApplicationManagerImpl()
                                       &ApplicationManagerImpl::OnTimerSendTTSGlobalProperties,
                                       true),
     is_low_voltage_(false) {
+
+    devicesType.insert(
+        std::make_pair(std::string("USB_AOA"), hmi_apis::Common_TransportType::USB_AOA));
+    devicesType.insert(
+        std::make_pair(std::string("USB_IOS"), hmi_apis::Common_TransportType::USB_IOS));
+    devicesType.insert(
+        std::make_pair(std::string("BLUETOOTH"), hmi_apis::Common_TransportType::BLUETOOTH));
+    devicesType.insert(
+        std::make_pair(std::string("WIFI"), hmi_apis::Common_TransportType::WIFI));
+
     std::srand(std::time(0));
     AddPolicyObserver(this);
 
-    dir_type_to_string_map_ = {
-      {TYPE_STORAGE, "Storage"},
-      {TYPE_SYSTEM, "System"},
-      {TYPE_ICONS, "Icons"}
-    };
+    dir_type_to_string_map_.insert(std::make_pair(TYPE_STORAGE, "Storage"));
```

Comment:
May be create method, which return this map?
It will allow to make *dir_type_to_string* as a const and help to avoid modification problems.